pom.xml

```diff
@@ -37,6 +44,7 @@
     <version.java>1.8</version.java>
     <maven.version>3.0</maven.version>
 
+    <detekt-maven-plugin.version>1.22.0</detekt-maven-plugin.version>
```

Comment:
Please, move it after the 73 line. Right now there is no clear structure, but I believe I will introduce it one day 🤞